mesheryctl/internal/cli/root/registry/generate.go

```diff
@@ -71,13 +71,16 @@ var generateCmd = &cobra.Command{
 	Long:  "Prerequisite: Excecute this command from the root of a meshery/meshery repo fork.\n\nGiven a Google Sheet with a list of model names and source locations, generate models and components any Registrant (e.g. GitHub, Artifact Hub) repositories.\n\nGenerated Model files are written to local filesystem under `/server/models/<model-name>`.",
 	Example: `
 // Generate Meshery Models from a Google Spreadsheet (i.e. "Meshery Integrations" spreadsheet).
-mesheryctl registry generate --spreadsheet-id "1DZHnzxYWOlJ69Oguz4LkRVTFM79kC2tuvdwizOJmeMw" --spreadsheet-cred 
+mesheryctl registry generate --spreadsheet-id "1DZHnzxYWOlJ69Oguz4LkRVTFM79kC2tuvdwizOJmeMw" --spreadsheet-cred
 // Directly generate models from one of the supported registrants by using Registrant Connection Definition and (optional) Registrant Credential Definition
 mesheryctl registry generate --registrant-def [path to connection definition] --registrant-cred [path to credential definition]
 // Generate a specific Model from a Google Spreadsheet (i.e. "Meshery Integrations" spreadsheet).
 mesheryctl registry generate --spreadsheet-id "1DZHnzxYWOlJ69Oguz4LkRVTFM79kC2tuvdwizOJmeMw" --spreadsheet-cred --model "[model-name]"
-// Generate Meshery Models and Component from csv files in a local directory.
-mesheryctl registry generate -directory <DIRECTORY_PATH>
+// Easily generate Meshery models and components from CSV files in a local directory using the following command.
+// To test, download each CSV file individually from GitHub. First, navigate to the [model and component CSV templates](https://github.com/meshery/meshery/tree/master/docs/assets/model-and-component-csv-templates), click on each file (model.csv and component.csv), and download them. Create a folder on your local machine, store both CSV files there, and use the command above with your local folder path.
```

Comment:
Looks good! Maybe add a note suggesting users customize the files for the specific model/component they’re contributing, just for added clarity. The rest is all nice!